go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/fatih/color v1.18.0
 	github.com/google/go-cmp v0.7.0
 	github.com/kong/go-apiops v0.1.41
-	github.com/kong/go-database-reconciler v1.22.2
+	github.com/kong/go-database-reconciler v1.22.3-0.20250407062325-f0a9ebb30d1b
```

Comment:
To be updated once https://github.com/Kong/go-database-reconciler/pull/242 is merged